mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -708,6 +708,105 @@ func TestClientSideEncryptionProse(t *testing.T) {
 			})
 		}
 	})
+	changeStreamOpts := mtest.NewOptions().
+		CreateClient(false).
+		Topologies(mtest.ReplicaSet)
+	mt.RunOpts("change streams", changeStreamOpts, func(mt *mtest.T) {
+		// Change streams can't easily fit into the spec test format because of their tailable nature, so there are two
+		// prose tests for them instead:
+		//
+		// 1. Auto-encryption errors for Watch operations. Collection-level change streams error because the
```

Comment:
I didn't realize we had all these restrictions, and that bypassAutoEncryption is required.